### PR TITLE
Some docstring improvements

### LINKED ISSUE #220: Fix broken docstrings
Part of SDI #205 , we want to fix some broken docstrings and improve the docs

### PR DESCRIPTION
**Pull request by valeriabarra**
## Purpose 
This PR addresses #220 , part of SDI #205 

Will close #220 

## To-do
- [x] Fix broken docstrings
- [x] Improve some math typesetting
- [x] Improve some other typesetting (e.g., code)

## Content
All of the above in the To-do list

I have:
- followed the codebase contribution guide: https://clima.github.io/ClimateMachine.jl/latest/Contributing/
- followed the style guide: https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/
- followed the documentation policy: https://github.com/CliMA/policies/wiki/Documentation-Policy
- checked that this PR does not duplicate an open PR.

In the Content, I have included 
- relevant unit tests, and integration tests, 
- appropriate docstrings on all functions, structs, and modules, and included relevant documentation.

- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/TimeManager.jl [hunk 1]
```diff
@@ -1,9 +1,9 @@
-#=
+"""
     TimeManager
 
 This module facilitates calendar functions and temporal interpolations
 of data.
-=#
+"""
```
Comment (charleskawczynski) [inline src/TimeManager.jl:6]: This is still broken due to the newline between the triple quotes and the module

File: src/ConservationChecker.jl [hunk 2]
```diff
@@ -147,15 +147,15 @@ function check_conservation!(
...
+    check_conservation!(
+    cc::WaterConservationCheck,
+    coupler_sim,
+    get_slab_energy,
+    get_land_energy,
+    )
 
-computes the total water ∫ ρq_tot dV of the various components
-of the coupled simulations, and updates cc with the values.
+computes the total water, ``\\int \\rho q_{tot} dV``, of the various components
```
Comment (charleskawczynski) [inline src/ConservationChecker.jl:157]: I actually prefer the existing format, at least for small doc strings like this-- it reads a bit simpler

**Review by charleskawczynski — APPROVED** [review]

File: src/TimeManager.jl (hunk 1 above)
Comment (valeriabarra) [reply src/TimeManager.jl:6]: Oh, thank you. I was checking the local build and did not realize. Thanks

File: src/ConservationChecker.jl (hunk 2 above)
Comment (valeriabarra) [reply src/ConservationChecker.jl:157]: I prefer the math typesetting :)  What does @LenkaNovak prefer?

Comment (LenkaNovak) [reply src/ConservationChecker.jl:157]: I agree with Charlie. The original is more readable from the REPL.

Comment (valeriabarra) [reply src/ConservationChecker.jl:157]: Oh, ok. I tend to look at the web version and I think the math typesetting makes it easier to read. It is ok, I can revert this change in a future PR

Comment (valeriabarra) [reply src/ConservationChecker.jl:157]: Done in #223

**Pull request merged**